packages/website/package.json

```diff
@@ -35,32 +35,21 @@
     "@typescript-eslint/eslint-plugin": "^2.33.0",
     "@typescript-eslint/parser": "^2.34.0",
     "@vue/cli": "^4.3.1",
-    "@vue/cli-plugin-babel": "^4.3.1",
     "@vue/cli-plugin-eslint": "^4.3.1",
     "@vue/cli-plugin-typescript": "^4.3.1",
     "@vue/cli-service": "^4.3.1",
-    "@vue/component-compiler-utils": "^3.1.2",
     "@vue/eslint-config-airbnb": "^5.0.2",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^5.0.1",
-    "autoprefixer": "^9.8.0",
-    "babel-plugin-transform-vue-jsx": "^3.7.0",
-    "core-js": "^3.6.5",
```

Comment:
This resolves the problem of some libraries defaulting to v2, it may now be resolved though.

Reply:
A quick search in `yarn.lock` shows that none of the dependencies declares core-js as a peerDep, so nothing will be default to v2 unless they explicitly want v2.

Reply:
See #145 and #184.

It seems the conflicts are now resolved.